Xamarin.PropertyEditing/Reflection/ReflectionEnumPropertyInfo.cs

```diff
@@ -65,7 +65,7 @@ public override void SetValue<TValue> (object target, TValue value)
 			}
 		}
 
-		public override TValue GetValue<TValue> (object target)
+		public override async Task<TValue> GetValueAsync<TValue> (object target)
```

Comment:
That's intentional. The `Reflection/` code is all just for testing, so it doesn't need to be all that great, this was just the quickest way of going from not building to building instead of sprinkling `Task.FromResult` everywhere.